xbmc/pictures/Picture.h

```diff
@@ -20,6 +20,7 @@ class CPicture
 {
 public:
   static bool GetThumbnailFromSurface(const unsigned char* buffer, int width, int height, int stride, const std::string &thumbFile, uint8_t* &result, size_t& result_size);
+  static bool CreateBase64ThumbnailFromSurface(const unsigned char *buffer, int width, int height, int stride, const std::string &thumbFile, std::string &base64buffer);
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types